bears/stylus/StylintBear.py

```diff
@@ -36,12 +38,182 @@ class StylintBear:
+        :param allow_colons:
+            When ``always``, expect ``:`` when declaring a property. When
+            ``never``, expect no ``:`` when declaring a property.
+        :param allow_colors:
+            When ``always``, enforce variables when defining hex values.
+        :param allow_spaces_after_commas:
+            Enforce or disallow spaces after commas.
+        :param allow_spaces_after_comments:
+            Enforce or disallow spaces after line comments.
+        :param allow_trailing_whitespace:
+            If ``false``, ignores trailing white space. If ``never``, trailing
+            white space will throw a warning.
+        :param allow_css_literal:
+            By default Stylint ignores ``@css`` blocks. If set to true however,
+            it will throw a warning if ``@css`` is used.
+        :param max_selector_depth:
+            Set the max selector depth. If set to 4, max selector depth will
+            be 4 indents. Pseudo selectors like ``&:first-child`` or
+            ``&:hover`` won't count towards the limit.
+        :param check_duplicates:
+            Checks if selector or property duplicated unnecessarily. By
+            default, only checks on a file-by-file basis, but if
+            ``globalDupes: true`` is set, then it will also check for
+            duplicates globally (for root values only).
+        :param allow_efficient_properties:
+            Check for places where properties can be written more efficiently.
+        :param set_extend_preference:
+            Pass in either ``@extend`` or ``@extends`` and then enforce that.
+            Both are valid in Stylus. It doesn't really matter which one
+            you use.
+        :param indent_size:
+            This works in conjunction with ``max_selector_depth``. If you
+            indent with spaces this is the number of spaces you indent with.
+            If you use hard tabs, set this value to ``false``.
+        :param allow_leading_zero:
+            Enforce or disallow unnecessary leading zeroes on decimal points.
+        :param set_max_errors:
+            Set 'max' number of Errors.
+        :param set_max_warnings:
+            Set 'max' number of Warnings.
+        :param mixed_spaces_and_tabs:
+            Returns ``true`` if mixed spaces and tabs are found. If a number is
+            passed to indent_size, it assumes soft tabs (ie, spaces)
+            , and if ``false`` is passed to indentPref it assumes hard tabs.
+        :param set_naming_convention:
+            Enforce a particular naming convention when declaring classes, ids,
+            and variables. Throws a warning if you don't follow the convention.
+        :param set_strict_naming_convention:
+            By default, ``set_naming_convention`` only looks at variable names.
+            If ``set_strict_naming_convention`` is set to true,
+            ``set_naming_convention`` will  also look at class and id names.
```

Comment:
rename the setting to resemble this better. I wouldn't have thought that the meaning of "strict" means to also check classes.
Can we have something like `variable_naming_convention` and `class_naming_convention`.